Library/Formula/nave.rb

```diff
@@ -0,0 +1,13 @@
+class Nave < Formula
+  homepage "https://github.com/isaacs/nave"
+  url "https://github.com/isaacs/nave/archive/v0.4.5.tar.gz"
+  sha1 "190fdab025882a9a50d85c2509b04f269f97dbf7"
+
+  def install
+    bin.install "nave.sh" => "nave"
+  end
+
+  test do
+    assert_match /Usage: nave/, shell_output("#{bin}/nave help")
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!